client/app/bundles/course/level/containers/LevelsLayout/index.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+import NotificationPopup from 'lib/containers/NotificationPopup';
+import Level from 'course/level/pages/Level';
+
+const LevelsLayout = () => (
```

Comment:
Actually can skip this file because this is used mainly for doing some routing between React components/pages, which is not required as there is only 1 page.